gst/nnstreamer/tensor_query/tensor_query_common.h

```diff
@@ -74,33 +79,63 @@ typedef struct
   };
 } TensorQueryCommandData;
 
-/**
- * @brief generate unique id.
- * @return unique id if OK, 0 if error
- */
-extern uint64_t
-nnstreamer_query_request_id (const char *ip, uint32_t port, int is_client);
-
 /**
  * @brief connect to the specified address.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_connect (uint64_t id, const char *ip, uint32_t port, uint32_t timeout_ms);
+nnstreamer_query_connect (query_connection_handle connection, const char *ip, uint32_t port, uint32_t timeout_ms);
 
 /**
  * @brief send command to connected device.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_send (uint64_t id, TensorQueryCommandData *data, uint32_t timeout_ms);
+nnstreamer_query_send (query_connection_handle connection, TensorQueryCommandData *data, uint32_t timeout_ms);
 
 /**
  * @brief receive command from connected device.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_receive (uint64_t id, TensorQueryCommandData *data, uint32_t timeout_ms);
+nnstreamer_query_receive (query_connection_handle connection, TensorQueryCommandData *data, uint32_t timeout_ms);
+
+/**
+ * @brief close connection with corresponding id.
+ * @return 0 if OK, negative value if error
+ */
+extern int
+nnstreamer_query_close (query_connection_handle connection);
+
+/* server */
+/**
+ * @brief accept client connection
+ * @param server_data TensorQueryServerData
```

Comment:
Which one is right between query_server_handle and TensorQueryServerData? and Please write more details for the doxygen tags.

Reply:
Fixed to `query_server_handle`. Thanks